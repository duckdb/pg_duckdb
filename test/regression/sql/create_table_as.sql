\set pwd `pwd`
CREATE TABLE webpages AS SELECT * FROM read_csv(:'pwd' || '/data/web_page.csv') as (column00 int, column01 text, column02 date);

select * from webpages order by column00 limit 2;
select count(*) from webpages;

DROP TABLE webpages;
