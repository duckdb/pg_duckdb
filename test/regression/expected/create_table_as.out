\set pwd `pwd`
CREATE TABLE webpages AS SELECT * FROM read_csv(:'pwd' || '/data/web_page.csv') as (column00 int, column01 text, column02 date);
select * from webpages order by column00 limit 2;
 column00 |     column01     |  column02  
----------+------------------+------------
        1 | AAAAAAAABAAAAAAA | 09-03-1997
        2 | AAAAAAAACAAAAAAA | 09-03-1997
(2 rows)

select count(*) from webpages;
 count 
-------
    60
(1 row)

DROP TABLE webpages;
